engine/src/main/java/com/datatorrent/stram/engine/StreamingContainer.java

```diff
@@ -7,7 +7,7 @@
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
  *
- *   http://www.apache.org/licenses/LICENSE-2.0
```

Comment:
Cannot modify license headers. License headers need to be consistent across all file types and comply with automated license check. This is not just an IDE concern.